### DISCUSSION TITLE
My dracula theme not working

### DISCUSSION THREAD

**Discussion by trungphanthanh** [dracula/dracula-theme#777] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49730157/202359264-3a62d1d1-47ca-4f29-ab59-09ed6a30cf54.png)
Please help my dracula theme not working with my text

**Comment by luxonauta** [comment, marked as answer]
🤔 That's strange; the most likely thing is that there are settings overriding theme changes. Try reinstalling it on a clean VSCode install!

**Reply by trungphanthanh** [reply]
i remove all my extensions and it working :v
